### ISSUE TITLE
Agents running in k8s should log a backtrace upon crashing

### ISSUE THREAD

**Issue by tkporter** [abacus-network/abacus-monorepo#624]
We've recently had some difficulty in debugging crashing agents that may be made easier by re-enabling the backtrace logs upon the agents crashing. Iirc, the reason they were taken out originally was because it created a big mess in GCP logging because they weren't in a nice single-line format and were instead split across many log lines that GCP logging didn't handle well. I believe one middle ground may be what was also suggested previously, which is to keep backtrace logs, but to log them without any newlines. It's probably best to have logs that are harder to decipher but may be helpful than no logs at all

**Comment by mattiecnvr** [comment]
Yeah, so I had plans to do this, but it wasn't super trivial and it was not a priority at the time.

**Comment by mattiecnvr** [comment]
Also this was changed by #412
